src/library/export/coverartcopywizard.h

```diff
@@ -0,0 +1,36 @@
+// CoverArtCopyWizard handles copying the cover art of the track to a where track located.
+
+#pragma once
+
+#include <QScopedPointer>
+#include <QString>
+
+#include "library/export/coverartcopyworker.h"
+#include "library/export/dlgcoverartcopy.h"
+#include "preferences/usersettings.h"
+
+class CoverArtCopyWizard : public QObject {
+    Q_OBJECT
+  public:
+    CoverArtCopyWizard(QWidget* parent,
+            UserSettingsPointer pConfig,
+            const QImage& coverArtImage,
+            const QString& coverArtCopyPath)
+            : m_parent(parent),
```

Comment:
The parent needs to be passed to the QObject constructor. The member variable can be disposed.
Use `parent()` instead.

Reply:
Member variable used on [here](https://github.com/mixxxdj/mixxx/blob/b9e082c2f8285ea11dec7ea0980f66081d3b33e9/src/library/export/coverartcopywizard.cpp#L5). If passing parent to QObject constructor needed, should it be also adjusted as well?